tests/openvino/native/test_model_transformer.py

```diff
@@ -162,3 +165,42 @@ def test_fq_insertion_weights(target_layers, ref_fq_names):
     assert len(fq_nodes) == len(ref_fq_names)
     for fq_name in fq_nodes:
         assert fq_name in ref_fq_names
+
+
+MODELS_WITH_PARAMETERS = [
+    {
+        'model': ConvModel().ov_model,
+        'layers': ['Conv'],
+        'values': [np.full((3,), 2)],
+        'refs': [2.0],
+    },
+    {
+        'model': FPModel(precision='FP16').ov_model,
+        'layers': ['MatMul'],
+        'values': [np.full((3,), 2)],
+        'refs': [2.0],
+    }
+]
+
+
+@pytest.mark.parametrize('model_with_parameters', MODELS_WITH_PARAMETERS)
+def test_bias_correction(model_with_parameters):
+    model = model_with_parameters['model']
+    layers = model_with_parameters['layers']
+    values = model_with_parameters['values']
+    refs = model_with_parameters['refs']
+
+    transformed_model = create_transformed_model(model, layers, TargetType.LAYER,
+                                                 OVBiasCorrectionCommand, port_id=1, **{'bias_value': values})
+    ops_dict = {op.get_friendly_name(): op for op in transformed_model.get_ops()}
+
+    for node_name, bias_reference in zip(layers, refs):
+        node = ops_dict[node_name]
+        node_inputs = [port.get_node() for port in node.output(0).get_target_inputs()]
+        node_with_bias = node_inputs[0]
+
+        potential_bias = node_with_bias.input_value(1).node
+        if potential_bias.get_type_name() == 'Convert':
+            potential_bias = potential_bias.input_value(0).node
+        assert potential_bias.get_type_name() == 'Constant'
+        assert np.mean(potential_bias.get_vector()) == bias_reference
```

Comment:
Why do you check here mean value and not the exact matching between tensors?

Reply:
Would it have any effect, while we are filling the bias constant with the known value (not random)? I think, it wouldn't.

Reply:
Anyway it confuses

Reply:
How is it confusing? What else can we assume here?

Reply:
It is better to check that every value is according to the reference, `mean` is extra computation here. 

It confuses because when you initially looked you spend some time to get why you need mean here. It is not what you expect - you expect to check tensor with tensor, not the mean value of tensor